src/db/tables/subscriber_email_preferences.ts

```diff
@@ -241,6 +248,6 @@ export {
   addEmailPreferenceForSubscriber,
   updateEmailPreferenceForSubscriber,
   getEmailPreferenceForSubscriber,
-  getEmailPreferenceForPrimaryEmail,
-  unsubscribeMonthlyMonitorReportForEmail,
+  getEmailPreferenceForUnsubscribeToken,
+  unsubscribeMonthlyMonitorReportForUnsubscribeToken,
 };
```

Comment:
Added unique index to the unsub token in the first PR. The index is not necessary anymore